src/main/resources/de/tracetronic/jenkins/plugins/ecutestexecution/configs/TestConfig/config.properties

```diff
@@ -1,4 +1,7 @@
 configSection.title=Test Configuration
+configSection.description=It is recommended to choose one of the following options:<br/><br/> \
+<li><b>Load Configuration:</b> Both the Test Bench Configuration and Test Configuration must be provided to load new configurations.</li> \
+<li><b>Keep Configuration:</b> Enabling this option retains the existing configuration for continued use throughout the execution.</li>
```

Comment:
* each information has to be related to its element (radio block)
  * use help attribute instead
*  I do not really have to define both configuration